### PR TITLE
Update coding standard to reflect changes in #1919

### PR DESCRIPTION
**Pull request by NathanJPhillips**
See #1919.

### CODE REVIEW

**Review by smowton — APPROVED** [review]

File: CODING_STANDARD.md [hunk 1]
```diff
@@ -48,8 +48,11 @@ Formatting is enforced using clang-format. For more information about this, see
 # Comments
 - Do not use `/* */`
 - Each source and header file must start with a comment block stating the
-  author. See existing source for an example of the format of this block. This
-  should be followed by a Doxygen `\file` comment:
+  author in this format:
+  ```c++
+  /// Author: Diffblue Ltd.
```
Comment (peterschrammel) [inline CODING_STANDARD.md:53]: I'd rather say ` /// Author: Your Name`, because this is a guideline for all public contributors.

**Comment by martin-cs** [comment]
+1 what Peter said.

**Review by kroening — CHANGES_REQUESTED** [review]
+1 Peter's comment

**Comment by NathanJPhillips** [comment]
Good point.

**Pull request closed without merging**